src/config_generator/snapshots/tailcall__config_generator__from_proto__test__required_types.snap

```diff
@@ -0,0 +1,18 @@
+---
+source: src/config_generator/from_proto.rs
+expression: cfg
+---
+schema @server @upstream {
+}
```

Comment:
query is missing here? there are no methods either.

Reply:
Because proto file doesn't have any methods. This was intentional